sdk/scheduler/src/main/java/com/mesosphere/sdk/curator/CuratorPersister.java

```diff
@@ -198,6 +198,94 @@ public static Builder newBuilder(String serviceName, String zookeeperHostPort) {
         }
     }
 
+    @Override
```

Comment:
The only new method I added is the `recursiveCopy` method in this file. The other methods were only reordered to be consistent with the interface and visibility.